grpc.go

```diff
@@ -1320,13 +1321,21 @@ func (a *agentGRPC) ReseedRandomDev(ctx context.Context, req *pb.ReseedRandomDev
 func (a *agentGRPC) GetGuestDetails(ctx context.Context, req *pb.GuestDetailsRequest) (*pb.GuestDetailsResponse, error) {
 	var details pb.GuestDetailsResponse
 	if req.MemBlockSize {
-		data, err := ioutil.ReadFile("/sys/devices/system/memory/block_size_bytes")
+		data, err := ioutil.ReadFile(sysfsMemoryBlockSizePath)
 		if err != nil {
-			return nil, err
-		}
-		details.MemBlockSizeBytes, err = strconv.ParseUint(string(data[:len(data)-1]), 16, 64)
-		if err != nil {
-			return nil, err
+			if os.IsNotExist(err) {
+				agentLog.WithField("sysfsMemoryBlockSizePath", sysfsMemoryBlockSizePath).Info("Guest kernel config doesn't support memory hotplug")
```

Comment:
Do you have a counter part PR to the runtime repo that handles zero memory block size returned by the agent?

Reply:
nice!